user/transmission/template.py

```diff
@@ -0,0 +1,83 @@
+pkgname = "transmission"
+pkgver = "4.0.6"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DENABLE_DAEMON=ON",
+    "-DENABLE_GTK=ON",
+    "-DENABLE_QT=ON",
+    "-DENABLE_CLI=ON",
+    "-DINSTALL_LIB=ON",
+    "-DUSE_SYSTEM_DEFLATE=ON",
+    "-DUSE_SYSTEM_EVENT2=ON",
+    "-DUSE_SYSTEM_MINIUPNPC=ON",
+    "-DUSE_SYSTEM_PSL=ON",
+    "-DUSE_SYSTEM_GTEST=ON",
+]
+make_check_args = ["-E", "LT.DhtTest.usesBootstrapFile"]
+hostmakedepends = ["cmake", "gettext", "libxml2-progs", "ninja", "pkgconf"]
+makedepends = [
+    "gtest-devel",
+    "gtkmm-devel",
+    "libcurl-devel",
+    "libdeflate-devel",
+    "libevent-devel",
+    "libpsl-devel",
+    "linux-headers",
+    "miniupnpc-devel",
+    "qt6-qtbase-devel",
+    "qt6-qtsvg-devel",
+    "qt6-qttools-devel",
+]
+pkgdesc = "BitTorrent client"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-2.0-or-later"
+url = "https://github.com/transmission/transmission"
+source = f"{url}/releases/download/{pkgver}/transmission-{pkgver}.tar.xz"
+sha256 = "2a38fe6d8a23991680b691c277a335f8875bdeca2b97c6b26b598bc9c7b0c45f"
+
+
+def post_install(self):
+    self.install_sysusers(self.files_path / "sysusers.conf")
+    self.install_tmpfiles(self.files_path / "tmpfiles.conf")
+    self.install_service(self.files_path / "transmission-daemon")
+
+
+@subpackage("transmission-qt")
+def _(self):
+    self.depends = [self.parent]
```

Comment:
these parent deps seem wrong, since the parent is the daemon and these are standalone applications that don't need the daemon (right?)

at best the base package is currently retaining the usr/share/icons that the desktop files in the -gtk -qt need, so not sure what i'd do about it, prolly add a -daemon for the daemon (and take the daemon and sysusers/tmpfiles there), and also add a -libs so it's not in base package, and then leave this and the gtk self.parent